packages/contracts/test/ScriptConfiguration.t.sol

```diff
@@ -23,23 +23,17 @@ contract ScriptConfiguration_Test is Test, Sphinx, SphinxTestUtils {
     }
 
     function configureSphinx() public override {
-        sphinxConfig.projectName = "test_project";
-        sphinxConfig.threshold = 1;
-        sphinxConfig.orgId = "test-org-id";
-        sphinxConfig.owners = [0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266];
+        sphinxConfig.projectName = "Simple_Project";
```

Comment:
To keep the `sphinx.lock` file simple, all our tests use a single project `Simple_Project`.